Translation/ConfigFactory.php

```diff
@@ -43,11 +57,20 @@ public function getBuilder($name)
         return $this->builders[$name];
     }
 
+    /**
+     * @param $name
+     * @param $locale
+     * @return mixed
+     */
     public function getConfig($name, $locale)
     {
         return $this->getBuilder($name)->setLocale($locale)->getConfig();
```

Comment:
I'm not sure this affects the PhpParser\Builder. The builders are an array of ConfigBuilder instances. Did I miss something? From JMSTranslationExtension.php